modules/graphdb/templates/02_dns_provisioning.sh.tpl

```diff
@@ -7,31 +7,76 @@
 # * Update the Route 53 hosted zone with an A record for the node's DNS name pointing to its local IPv4 address.
 # * Set the hostname of the EC2 instance to the newly created node DNS name.
 
-set -o errexit
+# set -o errexit is intentionally not used as the aws cli throws an error if a DNS record exists and it breaks the loop.
```

Comment:
Can't we check if it exists and delete/update? I think it's good to have `errexit`.

Reply:
There already is a check if the record is taken or not, however the record might be created between the check and the create command. Will think of another solution.

Reply:
Ok, found a way to work with `errexit`